src/solver/conv_asm_implicit_gemm_gtc_fwd_nhwc.cpp

```diff
@@ -312,12 +312,28 @@ GetFwdXdlopsNHWCConfigList()
     return kernel_param_list;
 }
 
-static std::tuple<std::string, // kernel_name
-                  size_t,      // block_size
-                  size_t,      // grid_size
-                  size_t>      // splits_4G
+// clang-format off
+static inline PerformanceConfigAsmImplicitGemmGTCFwdXdlopsNHWC
+GetFwdXdlopsNHWCConfigLargestTileFp32()
```

Comment:
Actually the fp32 largest tile is `256x128x16`, fp16/bf15 is `256x256x32`, there is still a difference. In case in the future need to change the largest tile size per different data type, I think it make sense to keep it as now

Reply:
`256x(512/datasize)x(64/datasize)`?

Reply:
I think to make a formula to calculate the tile size is intuitive, because tile size (a specific tuning parameter) does not have something to do with the data size.
Which means, if in the future, fp32 can provide a larger tile size like `256x192x16`, then the calculation will have wrong result

Reply:
I think the "calculation" approach make sense to host side maintainers, but for kernel developers there is really no relation ship between the largest tile size and the data type. It will confuse the kernel developers. And it's the kernel developer's responsibility to maintain the largest tile size table